LocalSettings.php

```diff
@@ -85,6 +85,10 @@
 define('NS_BLANK', 1662);
 define('NS_BLANK_TALK', 1663);
 
+// wisdomsandboxwiki
+define( 'NS_TEST',1632);
+define( 'NS_TEST_TALK',1633);
```

Comment:
Needs to be assigned/used in wgExtraNamespaces (near 1462)
https://github.com/miraheze/mw-config/blob/master/LocalSettings.php#L1462